test/app/App/Panel/AddToken/index.test.js

```diff
@@ -76,7 +76,7 @@ describe('token metadata lookup', () => {
       expect(tokenDecimalsInput.value).toEqual('18')
       expect(tokenChainSelect.textContent).toEqual('Mainnet')
     })
-  }, 1000)
+  }, 2000)
```

Comment:
are these tests regularly taking longer than 1 second? seems like something we should address as no individual unit test should be taking anywhere near this long

Reply:
> We'll probably need to ditch all the non-config changes and recreate the PR when it is time to merge anyway; the formatting changes in this PR are just to show what the formatted code would look like with these settings.

Reply:
When the tests run on every push, you know pretty quickly if they are flaky.